packages/stitch/tests/createDelegationPlanBuilder.test.ts

```
@@ -0,0 +1,144 @@
+              },
+            ],
+          },
+        },
+      ],
+    });
+
+    optimizeDelegationMap(delegationMap, 'User', fragments);
+    // Verify optimization results
+    expect(delegationMap.size).toBe(1);
+    expect(delegationMap.has(subschema2)).toBe(true);
+    const schema2Selections = delegationMap.get(subschema2)!.selections;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)